### PR TITLE
fix: change org open commands and flags to sf style

### PR DESCRIPTION
**Pull request by CristiCanizales**
### What does this PR do?
- Updates Org Open commands and flags to SF style in order to get rid of cli warnings

### What issues does this PR fix or reference?
@W-14047044@

### Changes
- `force:org:open` ⇢ `org:open`
- `--urlonly` ⇢ `--url-only`
- `SFDX_JSON_TO_STDOUT` ⇢ `SF_JSON_TO_STDOUT`

### Warnings before
```
18:54:02.495 sfdx force:org:open
(node:85263) Warning: Deprecated environment variable: SFDX_JSON_TO_STDOUT. Please use SF_JSON_TO_STDOUT instead.
(Use `node --trace-warnings ...` to show where the warning was created)
Warning: The "force org open" command has been deprecated. Use "org open" instead.
```

### CODE REVIEW

File: packages/salesforcedx-vscode-core/src/index.ts [hunk 1]
```diff
@@ -66,15 +65,14 @@ import {
   forceSourceRetrieveCmp,
   forceSourceRetrieveManifest,
   forceSourceRetrieveSourcePaths,
-  forceSourceStatus,
```
Comment (CristiCanizales) [inline packages/salesforcedx-vscode-core/src/index.ts:69]: this is not used in the file, so I removed it

File: packages/salesforcedx-vscode-core/src/index.ts [hunk 2]
```diff
@@ -66,15 +65,14 @@ import {
...
   forceStartApexDebugLogging,
   forceStopApexDebugLogging,
   forceTaskStop,
   forceVisualforceComponentCreate,
   forceVisualforcePageCreate,
   initSObjectDefinitions,
+  orgOpen,
   registerFunctionInvokeCodeLensProvider,
-  SourceStatusFlags,
```
Comment (CristiCanizales) [inline packages/salesforcedx-vscode-core/src/index.ts:77]: same here. Not used

File: packages/salesforcedx-vscode-core/src/index.ts [hunk 3]
```diff
@@ -123,12 +121,6 @@ import { OrgAuthInfo } from './util/authInfo';
 const flagOverwrite: FlagParameter<string> = {
   flag: '--forceoverwrite'
 };
-const flagStatusLocal: FlagParameter<SourceStatusFlags> = {
-  flag: SourceStatusFlags.Local
-};
-const flagStatusRemote: FlagParameter<SourceStatusFlags> = {
-  flag: SourceStatusFlags.Remote
-};
```
Comment (CristiCanizales) [inline packages/salesforcedx-vscode-core/src/index.ts:131]: same here with these 2 above. Not used

**Review by daphne-sfdc — APPROVED** [review]
Approved! 🎉

**Comment by daphne-sfdc** [comment]
Here are the steps that I have gone through to test that the changes in this PR have removed the org open warnings:

1. Create a new project.

2. Run “SFDX: Authorize an Org”.  Authorize my dev org.
a. Command Palette: Run “SFDX: Open Default Org”.  Make sure my dev org is open. ✅
b. Bottom Bar: Click the “Open Org” button.  Make sure my dev org is open. ✅

3. Run “SFDX: Authorize a Dev Hub”.  Authorize my dev org.

4. Run “SFDX: Create a Default Scratch Org”.
a. Command Palette: Run “SFDX: Open Default Org”.  Make sure the scratch org is open. ✅
b. Bottom Bar: Click the “Open Org” button.  Make sure the scratch org is open. ✅

5. Run “SFDX: Log Out from All Authorized Orgs”.
a. Command Palette: Run “SFDX: Open Default Org”.  No org should be opened, and there should be a notification that says “SFDX: Open Default Org failed to run”. ✅
b. Bottom Bar: Make sure the “Open Org” button is no longer present. ✅

While doing this testing, I did not see any warnings for the variables that were changed in this PR.  Therefore, the org open warnings are successfully removed in this code change.

**Pull request merged**